examples/express-es7/README.md

```diff
@@ -8,13 +8,6 @@ npm install
 npm start
 ```
 
-Note that starting from 0.8.0 objection uses native ES2015 classes and no longer supports
```

Comment:
Is it ok to remove this note after changing to the `env` preset? Or should I change it to inform users about that.

Reply:
Yes, I think this can be removed now.